src/components/media-views.js

```diff
@@ -199,10 +199,7 @@ function createVideoTexture(url, contentType) {
   });
 }
 
-function fitToTexture(el, texture) {
-  const ratio =
-    (texture.image.videoHeight || texture.image.height || 1.0) /
-    (texture.image.videoWidth || texture.image.width || 1.0);
+function scaleToAspectRatio(el, texture, ratio) {
```

Comment:
`texture` is unused. Can pass in height and width here like
`function scaleToAspectRatio(el, height, width)`
or remove this function and set the `scale` and set `matrixNeedsUpdate` at the call sites.

Reply:
Ah yeah woops. In fact we actually only care about ratio, so just going to store that instead